sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -62,17 +62,6 @@ class SQLContext private[sql](val sparkSession: SparkSession)
 
   sparkSession.sparkContext.assertNotStopped()
 
-  // Note: Since Spark 2.0 this class has become a wrapper of SparkSession, where the
```

Comment:
keep these two lines?

Reply:
Makes sense, will add it back. Yeah will leave this open a short while to make sure there is time to comment.